### PR TITLE
Add Nodemeta

### PR DESCRIPTION
**Pull request by Zeldhyr**
This add nodes meta data option (https://www.consul.io/api/catalog.html#nodemeta)

### CODE REVIEW

File: consul/resource_consul_node.go [hunk 1]
```diff
@@ -34,6 +34,12 @@ func resourceConsulNode() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"nodemeta": &schema.Schema{
```
[the remark this answers is not in the document]
Comment (paultyng) [reply consul/resource_consul_node.go]: It will need to be adjusted in the docs as well.

**Comment by pearkes** [comment]
Would be great to add tests to this @Zeldhyr. I can come back around and do so if you aren't able to get to it.

**Comment by Zeldhyr** [comment]
I do not have a lot of time. I would like some help for tests thx

**Comment by lawliet89** [comment]
@pearkes I have rebased this branch and updated the tests at my [fork](https://github.com/terraform-providers/terraform-provider-consul/compare/master...lawliet89:node-meta?expand=1). Should I open a PR to get that merged in?

**Comment by pearkes** [comment]
@lawliet89 yes, that looks good -- some minor doc comments but please open a PR and we can get that in!

**Comment by lawliet89** [comment]
Commits from https://github.com/terraform-providers/terraform-provider-consul/pull/65 which includes this PR has been merged. I think this PR can be closed.

**Comment by pearkes** [comment]
Agree @lawliet89 thanks.

**Pull request closed without merging**